### ISSUE TITLE
Options for muscle targets and myomatrix arrays

### ISSUE THREAD

**Issue by lambdaloop** [AllenNeuralDynamics/aind-data-schema#996]
I'm starting a new issue here to discuss how to specify myomatrix arrays and the muscles they target. 

Mafalda Vicente and I were thinking it could be something like the following for myomatrix array:

- number of branches
  - 4 branches 8 contacts currently
  - 8 branches 4 contacts eventually
  - even more Contacts?
- for each branch
  - number of contact in branch
  - muscle it's in

For the muscles, it could be an enum of the muscles allowed by our protocol to start and we can expand it as we add more muscles to the protocol.

**Comment by saskiad** [comment]
Surgery:
List of Arrays
array type (injected/sutured)
list of threads:
- Contacts: Body Part/Muscle/Notes/In Muscle (boolean)
- Ground electrode location (Body Part)

**Comment by lambdaloop** [comment]
See https://github.com/AllenNeuralDynamics/aind-data-schema-models/issues/37 for a discussion of the muscle model

**Comment by dbirman** [comment]
@saskiad did the code we wrote together for this get pushed? I was poking around to see where it was but I can't find the branch/PR

**Comment by saskiad** [comment]
not yet - I was waiting on the models thing to get merged first. I'll finish this today.
